src/main/java/com/redhat/rhjmc/containerjfr/net/web/WebServer.java

```diff
@@ -73,6 +74,12 @@
         this.savedRecordingsPath = savedRecordingsPath;
         this.logger = logger;
         this.reportGenerator = reportGenerator;
+
+        if (env.getEnv(USE_LOW_MEM_PRESSURE_STREAMING_ENV) != null) {
```

Comment:
I'd rather use `hasEnv` here than a null-check. Setting the variable to the empty string should still count as leaving it disabled, IMO. `hasEnv` checks for null-ness and blank-ness.

You could then also add a `-e` in run.sh for this variable.

Reply:
I'm aware of the existant of `.hasEnv()`, but I was thinking any non-null value, including an empty string, should be considered to enabling such feature.

But yes, I'll change it to use `.hasEnv()` and keep things consistent.